### DISCUSSION TITLE
Guide on how to make the localy self-hosted website avaiable on the internet?

### DISCUSSION THREAD

**Discussion by xMannyGamingx** [nfriedly/nodeunblocker.com#42] [Q&A]
Would be nice, Thanks.

-Manny

**Comment by nfriedly** [comment]
I don't think I'm going to write one, but perhaps I can link to one or two good ones.

**Comment by toydotgame** [comment]
You can google tutorials for how to set up a static IP connection in your LAN and also port forwarding. It's a very common thing; [portforward.com](https://portforward.com/) and [canyouseeme.org](https://canyouseeme.org) are useful resources for the port forwarding part.
